app/web/features/auth/signup/AccountForm.tsx

```diff
@@ -206,19 +216,35 @@ export default function AccountForm() {
           className={authClasses.formField}
           control={control}
           error={!!errors?.birthdate?.message}
-          helperText={errors?.birthdate?.message ?? " "}
+          helperText={errors?.birthdate?.message}
           id="birthdate"
           rules={{
             required: t("auth:account_form.birthday.required_error"),
-            validate: (stringDate: string) =>
-              validatePastDate(stringDate) ||
-              t("auth:account_form.birthday.validation_error"),
+            validate: (stringBirthDate: string) => {
+              const birthDate = dayjs(stringBirthDate);
+              const age = Math.abs(dayjs().diff(birthDate, "year"));
+
+              if (age < 18) {
+                return t("auth:account_form.birthday.too_young_error");
+              }
+
+              if (age > 120) {
+                return t("auth:account_form.birthday.validation_error");
+              }
```

Comment:
The oldest know human is 122 so 🤷‍♀️. Let me know if I should make it lower.

Reply:
Yeah I think the bound is fine, but maybe the error should say something else, we can just say "Please pick a real birthdate".